interface-definitions/service_ids_suricata.xml.in

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="service">
+    <children>
+      <node name="ids">
+        <children>
+          <node name="suricata" owner="${vyos_conf_scripts_dir}/service_ids_suricata.py">
+            <properties>
+              <help>Suricata high performance Network IDS, IPS and Network Security Monitoring engine</help>
```

Comment:
Please shorten to: `Network IDS, IPS and Network Security Monitoring`

The term `Suricata` is redundant as the node already carries the name. I'd also avoid patterns like "high performance" ;)